core/src/navigation/services/semi-collapsed-navigation.js

```diff
@@ -7,19 +7,23 @@ class SemiCollapsibleNavigationClass {
     this.responsiveNavSetting = LuigiConfig.getConfigValue(
       'settings.responsiveNavigation'
     );
-    let currentCollapsedState = localStorage.getItem(
-      NavigationHelpers.COL_NAV_KEY
+    let currentCollapsedState = JSON.parse(
```

Comment:
This should be `const` as well.
Is this a `boolean` ?    If so, could you please change to `is` prefix naming?

Reply:
it is a string, Hannes mentioned it above, when suggesting the changes

Reply:
This should be a boolean after `JSON.parse`

Reply:
right, mixed it up and now renamed to `isCurrentStateCollapsed`